cmd/influx/cli/cli_test.go

```diff
@@ -473,6 +494,48 @@ func TestParseCommand_HistoryWithBlankCommand(t *testing.T) {
 func emptyTestServer() *httptest.Server {
 	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Influxdb-Version", SERVER_VERSION)
-		return
+
+		switch r.URL.Path {
+		case "/query":
+			values := r.URL.Query()
+			parser := influxql.NewParser(bytes.NewBufferString(values.Get("q")))
+			q, err := parser.ParseQuery()
+			if err != nil {
+				w.WriteHeader(500)
```

Comment:
Can you use the status code constants in `net/http`? e.g. `http.StatusInternalServerError`, `http.StatusOK`